### PR TITLE
Add ligo-gracedb

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds [`ligo-gracedb`](https://pypi.org/project/ligo-gracedb/), a python client for the Gravitational-Wave Candidate Event DataBase.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
I would appreciate some help understanding why this build failed on appveyor. It seems the Windows install can't find/execute the script installed with this package. I can't see what's wrong, but then again I know ~nothing about Python on Windows.

**Comment by duncanmmacleod** [comment]
I think this is ready to go now.

**Comment by duncanmmacleod** [comment]
@jakirkham, @mariusvniekerk (I'm not sure who I'm supposed to ping if I'm supposed to ping anybody), is this ready for merging?

**Comment by duncanmmacleod** [comment]
@mariusvniekerk, @jakirkham, ping?

File: recipes/ligo-gracedb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
```
Comment (marcelotrevisani) [inline recipes/ligo-gracedb/meta.yaml]: Could you please just put a comment here explaining the reasons why you patched it ?

Comment (duncanmmacleod) [reply recipes/ligo-gracedb/meta.yaml]: @marcelotrevisani see d48895b

Comment (marcelotrevisani) [reply recipes/ligo-gracedb/meta.yaml]: Thanks! :)
Looks good to me, after CI I will merge that

**Comment by marcelotrevisani** [comment]
Thanks @duncanmmacleod to put this project in ``conda-forge`` !

**Pull request merged**